CONTRIBUTING.md

```diff
@@ -33,189 +33,16 @@ If your contribution is significantly big it is better to first check with the p
 
 Want to get started hacking on the code? Great! Keep reading. This contributing guide helps you to start working with and contributing to the Azure Typescript/Javascript SDK Generator.
 
-### General Autorest generation design
+### General Introduction
 
-_TBA_
+Inside this project, we support both high level client generation and rest level client generation. In the rest level client generation, we support generate code from both rest api specs and cadl. In the high level client generation, we only support generation from rest api specs, the support of generate high level client from cadl is planning.  
```

Comment:
not sure whether we need to mention  'the support of generate high level client from cadl is planning.'